enhancements/oc/must-gather.md

```diff
@@ -64,16 +64,20 @@ ship a second one.
 
 `must-gather` for openshift is a combination of three tools:
 
-1. A client-side `inspect` command that works like a super-get.  It has semantic understand of some resources and traverses
-   links to get interesting information beyond the current.  Pods in namespaces and logs for those pods as a for instance.
-   Currently this is `openshift-must-gather inspect`, but we are porting this to `oc adm` as experiemental in 4.3.  We may
+1. A client-side `inspect` command that works like a super-get. It has semantic understanding of some resources and traverses
+   links to get interesting information beyond the current. Pods in namespaces and logs for those pods, for instance.
+   Currently, this is `openshift-must-gather inspect`, but we are porting this to `oc adm` as experimental in 4.3. We may
    change and extend arguments over time, but the intent of the command will remain.
-2. The openshift-must-gather image, produced from https://github.com/openshift/must-gather.  The entry point is a
+2. The openshift-must-gather image, produced from https://github.com/openshift/must-gather. The entry point is a
    [/gather bash script](https://github.com/openshift/must-gather/blob/master/collection-scripts/gather) owned by CEE
```

Comment:
This is now owned by the workloads team, and/or the functional engineering teams for their respective product focus areas.

Reply:
Ack. I could update that in a separate PR along with https://github.com/openshift/enhancements/pull/1487#discussion_r1368358904.

Reply:
I'd let the workloads team update that after this merges.